app/api/v2/identity/general.rb

```diff
@@ -3,6 +3,19 @@
 module API::V2
   module Identity
     class General < Grape::API
+
+      desc 'KYCAID callback'
```

Comment:
Rename it KYC callback

Reply:
Can we allow any params and push them to the KYC plugin (validation must be done in the plugin itself)